### PR TITLE
clang-6/gcc: compiler barking fix

### PR DESCRIPTION
**Pull request by kyessenov**

### CODE REVIEW

File: src/envoy/http/mixer/control_factory.h [hunk 1]
```diff
@@ -82,6 +82,7 @@ class ControlFactory : public Logger::Loggable<Logger::Id::config> {
         case istio::utils::Logger::Level::ERROR_:
           return ENVOY_LOG_CHECK_LEVEL(error);
```
Comment (PiotrSikora) [inline src/envoy/http/mixer/control_factory.h]:
```
default:
  NOT_REACHED_GCOVR_EXCL_LINE;
```
is more in line with Envoy's coding style, and it should fix this as well.

Comment (kyessenov) [reply src/envoy/http/mixer/control_factory.h]: done

**Comment by duderino** [comment]
retroactive /lgtm

**Pull request merged**